docs/tutorials/csharp/stable-diffusion-csharp.md

```diff
@@ -0,0 +1,330 @@
+---
+title: Stable Diffusion with C#
+description: In this tutorial we will learn how to do inferencing for the popular Stable Diffusion deep learning model in C#.
+parent: Inference with C#
+grand_parent: Tutorials
+has_children: false
+nav_order: 1
+---
+
+# Inference Stable Diffusion with C# and ONNX Runtime
+{: .no_toc }
+
+In this tutorial we will learn how to do inferencing for the popular Stable Diffusion deep learning model in C#. Stable Diffusion models denoise a static image to create an image that represents the text prompt given by the user.
+
+```text
+"make a picture of green tree with flowers around it and a red sky" 
+```
+
+|----------------------|----------------------|
+| <img src="../../../images/latents-noise-example.png" width="256" height="256" alt="Image of browser inferencing on sample images."/> | <img src="../../../images/sample-output-stablediff.png" width="256" height="256" alt="Image of browser inferencing on sample images."/> |
+
+
+## Contents
+{: .no_toc }
+
+* TOC placeholder
+{:toc}
+
+## Prerequisites
+This tutorial can be run locally or in the cloud by leveraging Azure Machine Learning compute.
+
+- [Download the Source Code from GitHub](https://github.com/cassiebreviu/StableDiffusion)
+
+To run locally:
+
+- [Visual Studio](https://visualstudio.microsoft.com/downloads/) or [VS Code](https://code.visualstudio.com/Download)
+- A GPU enabled machine with CUDA EP Configured. This was built on a GTX 3070 and it has not been tested on anything smaller. Follow [this tutorial to configure CUDA and cuDNN for GPU with ONNX Runtime and C# on Windows 11](https://onnxruntime.ai/docs/tutorials/csharp/csharp-gpu.html)
+
+To run in the cloud with Azure Machine Learning:
+
+- [Azure Subscription](https://azure.microsoft.com/free/)
+- [Azure Machine Learning Resource](https://azure.microsoft.com/services/machine-learning/)
+
+## Use Hugging Face to download the Stable Diffusion models
+
+The Hugging Face site has a great library of open source models. We will leverage and download the [ONNX Stable Diffusion models from Hugging Face](https://huggingface.co/models?sort=downloads&search=Stable+Diffusion).
+
+ - [ONNX Models for v1.4](https://huggingface.co/CompVis/stable-diffusion-v1-4/tree/onnx)
```

Comment:
"ONNX Models for v1.4" made me think it was for ORT 1.4 or mybe some opset. 
Suggest renaming this and the next to "Stable Diffusion v1.4" and "Stable Diffusion v1.5"

Reply:
maybe its just me but the "for v1.4" is still making me think its for a specific ORT version or opset version. I think it's sufficient for the link to say "Stable Diffusion v1.4" but if you want to mention ONNX, then "ONNX models for Stable Diffusion v1.4" would be clearer.

Reply:
Fixed